desktop/core/src/desktop/lib/idbroker/conf.py

```diff
@@ -43,7 +44,16 @@ def validate_fs(fs=None):
 
 def get_cab_address(fs=None):
   fs = validate_fs(fs)
-  return get_conf().get(_CNF_CAB_ADDRESS % fs) if fs else None
+  if fs:
+    id_broker_addr = get_conf().get(_CNF_CAB_ADDRESS % fs)
+    if id_broker_addr:
+      id_broker_addr_list = id_broker_addr.split(',')
+      id_broker_rand = random.choice(id_broker_addr_list)
```

Comment:
Since id_broker_addr is not empty so the list always has one. We can just call id_broker_addr_list[0] instead import another new lib here?

Reply:
Isn't the idea behind the list to first try with the first entry, if it fails, try with the second and so on? Random is also OK I guess but will it now recover if one of them fails?